docs/resources/deployment.md

```diff
@@ -25,7 +25,7 @@ resource "prefect_flow" "flow" {
 }
 
 resource "prefect_deployment" "deployment" {
-  name                     = "%s"
+  name                     = "my-deployment"
```

Comment:
Weird, the TF docs job ran on the last commit in the related PR and said it had nothing to commit: https://github.com/PrefectHQ/terraform-provider-prefect/actions/runs/10397466852/job/28793213803

Guess it's catching up now.